cinema/serializers.py

```diff
@@ -125,9 +125,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
```

Comment:
Ensure that the `attrs` dictionary contains the keys 'row', 'seat', and 'movie_session' before accessing them. If any of these keys are missing, it could lead to a KeyError. Consider adding a check or using `attrs.get()` to handle missing keys gracefully.